app/panel/components/Settings/TrustAndRestrict.jsx

```diff
@@ -137,6 +137,35 @@ class TrustAndRestrict extends React.Component {
 			this.props.actions.updateSitePolicy({ type: otherListType, pageHost });
 		}
 		this.props.actions.updateSitePolicy({ type: listType, pageHost });
+		if (listType === 'whitelist') {
+			this.setState({ trustedValue: '' });
+		} else {
+			this.setState({ restrictedValue: '' });
+		}
+	}
+
+	isValidUrlorWildcard(pageHost) {
```

Comment:
This fails when trying to add `localhost:3000` to the list.

Reply:
Good catch! I fixed the validation regex and added a unit test